### ISSUE TITLE
Latest HA nodes crashing Node-RED on startup

### ISSUE THREAD

**Issue by liamhogan** [zachowj/node-red-contrib-home-assistant-websocket#1000]
### Describe the bug

After updating to the latest nodes (node-red-contrib-home-assistant-websocket 0.55.1), my Node-RED instance crashes on startup and gets stuck in a loop. 

I have isolated the issue to the HA server config node as when I remove that, Node-RED will startup correctly. 

The error I get in the log is not particularly informative even when logging level is set to trace: 
> `[error] UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "#<Object>".`

When I downgrade back to the previous version I was on (0.50.0), the crash doesn't happen.

Let me know if there's anywhere else I can look for info or if any further information is required. 

### This package's version (not the Home Assistant add-on version)

0.55.1

### Is Node-RED running in Docker?

Yes

### Node-RED version

3.0.2

### Node.js version

16.16.0

**Comment by zachowj** [comment]
You could try adding the below snippet to the top of your Node-RED settings file to try to get a better idea of what is not being caught.

```
process.on('unhandledRejection', (reason, p) => {
  console.log('Unhandled Rejection: ', p, 'reason:', reason);
});
```

Are you running the custom component, Node-RED companion? If so, is it updated to the latest version?

**Comment by liamhogan** [comment]
Thanks @zachowj. The return I got back from that is longer but doesn't seem much more helpful unfortunately. Here's the log snippet (redacted addresses):

> `[info] [server:*****] Connecting to http://*****`
`22 Jul 20:16:57 - [info] [server:*****] Connected to http://*****`
`22 Jul 20:16:57 - [debug] [server:*****] States Loaded`
`22 Jul 20:16:57 - [debug] [server:*****] HA State: running`
`22 Jul 20:16:57 - [debug] [server:*****] Integration: notloaded`
`Unhandled Rejection:  Promise {`
`  <rejected> { code: 'unknown_command', message: 'Unknown command.' }`
`} reason: { code: 'unknown_command', message: 'Unknown command.' }`

I am not running Node-RED companion.

**Comment by zachowj** [comment]
Could you temporarily load Node-RED companion to see if it stops the crashing? 

> but doesn't seem much more helpful unfortunately.

Actually, it was helpful.

**Comment by zachowj** [comment]
What version of Home Assistant are you running?

**Comment by liamhogan** [comment]
@zachowj I feel that loading additional software / components may complicate rather than resolve the situation. I've been using this node-red addon for a couple years now without the companion and with very few issues. 
That said, I did try to install the companion, but get an error in HA when trying to add the integration:

`Config flow could not be loaded: {"message":"Invalid handler specified"}`

I'm running Home Assistant Container (via docker) 2022.12.7

**Comment by liamhogan** [comment]
@zachowj has something been changed in these nodes since 0.50.0 that now requires the companion, or requires a core / standalone install of Home Assistant OS?

**Comment by liamhogan** [comment]
@zachowj
I updated my Home Assistant to the latest available version (2023.7.3) which seems to have fixed the issue. 

I noticed in your release notes that there is a breaking change from v0.55.0 which requires HA 2023.3.0 or later.
**The main page / readme of this repo is incorrect** and states the minimum HA version is v2021.3.0 

You need to update this and maybe add a reminder for this breaking change in subsequent recent releases.

**Comment by zachowj** [comment]
> I noticed in your release notes that there is a breaking change from v0.55.0 which requires HA 2023.3.0 or later.

Yes, I added that after I asked you what version of HA you were running. Version 0.55.0 is using a new [endpoint](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/commit/d630751c1274660150a85d8fa1d5243cc4096dc5) to stop from overloading the websocket when user has a large amount of entities. But this end point was only added to HA in v2023.3.0
